keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -106,66 +307,70 @@ type RollingUpdateStatefulSetStrategy struct {
   are the possible behavior choices we have:-
 
-  1. Pods with ordinal 4 and 3 will start Terminating at the same time(because of maxUnavailable). Once they are both running and ready, pods with ordinal 2 will start Terminating. Pods with ordinal 0 and 1 
-will remain untouched due the partition. In this choice, the number of pods terminating is not always 
-maxUnavailable, but sometimes less than that. For e.g. if pod with ordinal 3 is running and ready but 4 is not, we still wait for 4 to be running and ready before moving on to 2. This implementation avoids 
-out of order Terminations of pods.
-  2. Pods with ordinal 4 and 3 will start Terminating at the same time(because of maxUnavailable). When any of 4 or 3 are running and ready, pods with ordinal 2 will start Terminating. This could violate 
-ordering guarantees, since if 3 is running and ready, then both 4 and 2 are terminating at the same 
-time out of order. If 4 is running and ready, then both 3 and 2 are Terminating at the same time and no ordering guarantees are violated. This implementation, guarantees, that always there are maxUnavailable number of Pods Terminating except the last batch.
-  3. Pod with ordinal 4 and 3 will start Terminating at the same time(because of maxUnavailable). When 4 is running and ready, 2 will start Terminating. At this time both 2 and 3 are terminating. If 3 is 
-running and ready before 4, 2 wont start Terminating to preserve ordering semantics. So at this time, 
-only 1 is unavailable although we requested 2.
-  4. Introduce a field in Rolling Update, which decides whether we want maxUnavailable with ordering or without ordering guarantees. Depending on what the user wants, this Choice can either choose behavior 1 or 3 if ordering guarantees are needed or choose behavior 2 if they dont care. To simplify this further
-PodManagementPolicy today supports `OrderedReady` or `Parallel`. The `Parallel` mode only supports scale up and tear down of StatefulSets and currently doesnt apply to Rolling Updates. So instead of coming up
-with a new field, we could use the PodManagementPolicy to choose the behavior the User wants.
-
-        1. PMP=Parallel will now apply to RollingUpdate. This will choose behavior described in 2 above.
-           This means always maxUnavailable number of Pods are terminating at the same time except in 
-           the last case and no ordering guarantees are provided.
-        2. PMP=OrderedReady with maxUnavailable can choose one of behavior 1 or 3. 
-
-NOTE: The goal is faster updates of an application. In some cases , people would need both ordering 
-and faster updates. In other cases they just need faster updates and they dont care about ordering as 
+  1.  Pods with ordinal 4 and 3 will start Terminating at the same time(because of maxUnavailable). Once they are both running and available, pods with ordinal 2 will start Terminating. Pods with ordinal 0 and 1
```

Comment:
```suggestion
  1.  Pods with ordinal 4 and 3 will start Terminating at the same time (because of maxUnavailable). Once they are both running and available, pods with ordinal 2 will start Terminating. Pods with ordinal 0 and 1
```